src/pages/verify/e-ids/dutch-idin.mdx

```diff
@@ -0,0 +1,239 @@
+    sub: '{7d704c6d-4d12-45a0-ba8d-135d18f9d518}',
+    'http://schemas.grean.id/claims/sessionindex': '16494935-5ffc-4ae6-a683-6587a27b35c4',
+    'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/country': 'NL',
+    email: 'info@equensworldline.nl',
+    phone_number: '+31203051900',
+    'http://schemas.grean.id/claims/gender': 'MALE',
+    'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname': 'Vries',
+    family_name: 'Vries',
+    'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/dateofbirth': '1975-07-25',
+    birthdate: '1975-07-25',
+    verified_claims: {
+      bin: 'FANTASYBANK1234567890',
+      transientid: null,
+      legallastname: 'Vries',
+      preferredlastname: 'Vries-Jansen',
+      partnerlastname: 'Jansen',
+      legallastnameprefix: 'de',
+      preferredlastnameprefix: 'de',
+      partnerlastnameprefix: 'de',
+      initials: 'VJ',
+      dateofbirth: '1975-07-25',
+      ageCheck: null,
+      street: 'Pascalstreet',
+      houseno: '19',
+      housenosuf: null,
+      addressextra: null,
+      postalcode: '0000AA',
+      city: 'Aachen',
+      intaddressline1: null,
+      intaddressline2: null,
+      intaddressline3: null,
+      country: 'DE',
+      phone1: '+31203051900',
+      phone2: null,
+      phone3: null,
+    },
+    iat: 1697052782,
+    nbf: 1697052782,
+    exp: 1697055314,
+  }}
+/>
+
+### Re-identification
+
+Use `scope=idin:re-id`, or add `login_hint=scope:idin:re-id` to the authorize request to get this profile:
+
+<JwtPayloadViewer
+  payload={{
+    iss: 'https://localhost:44362',
+    aud: 'https://localhost:44301/',
+    nonce: 'ecnon',
+    identityscheme: 'nlidin',
+    authenticationtype: 'urn:grn:authn:nl:idin',
+    'http://schemas.microsoft.com/ws/2008/06/identity/claims/authenticationmethod': 'urn:grn:authn:nl:idin',
+    'http://schemas.microsoft.com/ws/2008/06/identity/claims/authenticationinstant': '2023-10-11T19:33:06.499Z',
+    'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier': '7d704c6d4d1245a0ba8d135d18f9d518',
+    sub: '{7d704c6d-4d12-45a0-ba8d-135d18f9d518}',
+    'http://schemas.grean.id/claims/sessionindex': '4c6fc8ad-9ddd-4176-ad2f-0d8a11ba9b88',
+    'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/country': 'NL',
+    verified_claims: {
+      bin: 'FANTASYBANK1234567890',
+      transientid: null,
+      legallastname: null,
+      preferredlastname: null,
+      partnerlastname: null,
+      legallastnameprefix: null,
+      preferredlastnameprefix: null,
+      partnerlastnameprefix: null,
+      initials: null,
+      dateofbirth: null,
+      ageCheck: null,
+      street: null,
+      houseno: null,
+      housenosuf: null,
+      addressextra: null,
+      postalcode: null,
+      city: null,
+      intaddressline1: null,
+      intaddressline2: null,
+      intaddressline3: null,
+      country: null,
+      phone1: null,
+      phone2: null,
+      phone3: null,
```

Comment:
I'm leaning towards leaving null-valued props out of the re-id, safe-login and age-verification, to be able to highlight just the relevant fields for each case.